cmd/vic-machine/configure/configure.go

```diff
@@ -221,6 +231,14 @@ func (c *Configure) copyChangedConf(o *config.VirtualContainerHostConfigSpec, n
 	if n.RegistryCertificateAuthorities != nil {
 		o.RegistryCertificateAuthorities = n.RegistryCertificateAuthorities
 	}
+
+	if n.InsecureRegistries != nil {
+		o.InsecureRegistries = n.InsecureRegistries
+	}
+
+	if n.RegistryWhitelist != nil {
+		o.RegistryWhitelist = n.RegistryWhitelist
+	}
```

Comment:
So actually I don't think I need to change this file at all. I'm going to remove the whitelist copy for the time being as to not increase the scope of this issue and maybe get it in either on time or barely late